nms/app/packages/magmalte/app/views/dashboard/feg/FEGDashboard.js

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import DashboardAlertTable from '../../../components/DashboardAlertTable';
+import EventAlertChart from '../../../components/EventAlertChart';
+import EventsTable from '../../events/EventsTable';
+import FEGDashboardKPIs from '../../../components/FEGDashboardKPIs';
+import Grid from '@material-ui/core/Grid';
+import React, {useState} from 'react';
+import Text from '../../../theme/design-system/Text';
+import TopBar from '../../../components/TopBar';
+import moment from 'moment';
+
+import {DateTimePicker} from '@material-ui/pickers';
+import {NetworkCheck} from '@material-ui/icons';
+import {Redirect, Route, Switch} from 'react-router-dom';
+import {colors} from '../../../theme/default';
+import {makeStyles} from '@material-ui/styles';
+import {useRouter} from '@fbcnms/ui/hooks';
+
+const EVENT_STREAM = 'NETWORK';
```

Comment:
Is there no central place to put this constant? This is likely used for different network types.

Reply:
Defined it and exported it from the events table because that is the place where it is created, which makes it the central place. Also, since this is only used as a prop for the EventsTable component, it is good to define it there.